README.md

```diff
@@ -18,10 +18,8 @@ For existing projects, AWS Classic remains fully supported; at this time, we rec
 
 ## Configuring credentials
 
-Credentials configuration is compatible with the classic AWS provider.
-
-Please refer to [this quickstart guide](
-https://www.pulumi.com/docs/intro/cloud-providers/aws-native/setup/) for possible configuration options.
+Credentials configuration is compatible with the classic AWS provider. Refer to the [AWS configuration options](
```

Comment:
Same note as in Azure-Native - the compatibility claim is somewhat loaded, I'd be happy to remove it.